### ISSUE TITLE
Clarify credentials need to be associated with certifier

### ISSUE THREAD

**Issue by mattgarrish** [w3c/epub-specs#1410]
The current definition of `certifierCredential` states:

> Identifies a credential or badge that establishes the authority of the party identified in the certifiedBy property to certify content accessible

This is too general a statement, as more than one certification can be listed, so more than one certifier can be listed, too.

We didn't use the `refines` attribute to tie the two together because it was expected to be deprecated in 3.1, but we should correct that now.

The definition should say "in the associated `certifiedBy` property" and the `refines` attribute should be added to the example.

/cc @GeorgeKerscher @avneeshsingh @clapierre

**Comment by clapierre** [comment]
Question, couldn't the certifierCredential or "badge" be an actual stamp/logo (ie. image) within the EPUB, along with a text description of the credential?  Right now its a text string which could be a URL or plain text of the credentials name.

This has become somewhat problematic with Vitalsource & Redshelf who wished to show this badge for those EPUBs which has this metadata.  Now out-of-band mechanism between a library / bookstore and the certifiedBy party is needed to provide this image with its description for those EPUBs with a specific certifierCredential.

**Comment by GeorgeKerscher** [comment]
If we do this, we would also need to provide the alt text. This is also a discussion right now in the WG for the primary metadata entry for the title.

**Comment by mattgarrish** [comment]
It's potentially overloading one piece of metadata to try and make it both a credential and a logo. A logo sounds like a separate piece of information.

There's also a question of whether a logo is something that belongs in the container, as wouldn't you want everyone to reference a version that is always up to date?

And once it's at least probable to be hosted outside the container, a manifest mechanism for alt text isn't going to help, since you can't reference remote images.

I'd imagine the credential name is probably your alt text in most cases, though. As to identifying a logo, that probably belongs in a `link` element. The question becomes how convoluted is it to express that there is a logo for a credential for a certifier. We're really pushing the boundaries of EPUB's lack of structured metadata.

It might help to cook up some examples of how this might work.

**Comment by mattgarrish** [comment]
Here's at least what I'm picturing:

```
<meta property="a11y:certifiedBy" id="cert">Ronald Ruck</meta>
<meta property="a11y:certifierCredential" id="cred" refines="#cert">A11y Credential</meta>
<link rel="icon" href="https://example.org/credential/a11y.svg" refines="#cred"/>
```

(Not sure if icon is the best rel, but was the first thing that came to mind.)

**Comment by clapierre** [comment]
This is what I was thinking as well Matt.
If the author wanted to include the icon inside the EPUB would it be

`<link rel="icon" href="../images/cover.jpg/a11y.svg" refines="#cred"/>`

**Comment by mattgarrish** [comment]
I'm going to open a new issue about images so this one can stay focused on adding `refines` back.

**Comment by mattgarrish** [comment]
Proposed Solution

Update the definitions for both certifierCredential and certifierReport to make clear that they are associated with the `certifiedBy` property and update the examples to include the `refines` attribute.

**Comment by clapierre** [comment]
>(Not sure if icon is the best rel, but was the first thing that came to mind.)

According to our marketing department usually these are called "Marks" or "Stamps". Icon can work too I suppose.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2020-12-10-epub-a11y#section3) on 2020-12-10)

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

### 3. Clarify credentials need to be associated with certifier 

_See github issue [#1410](https://github.com/w3c/epub-specs/issues/1410)._




**Avneesh Singh:** credentials refine certifier, but this relationship is not clear in the spec.  
… Matt G suggested that we formalize this relationship.  

> *Matthew Chan:* Matt G's example [https://github.com/w3c/epub-specs/issues/1410#issuecomment-724095290](https://github.com/w3c/epub-specs/issues/1410#issuecomment-724095290).

> **Proposed resolution: associate CertifiedBy with CertifierCredentials with refine.** *(Avneesh Singh)*

> *Matthew Chan:* +1.

> *Charles LaPierre:* +1.

> *Ben Schroeter:* +1.

> *Juliette McShane:* +1.

> *Bill Kasdorf:* +1.

> *Jun’Ichi Yoshii:* +1.

> *George Kerscher:* +1.

> *Laura Brady:* +1.

</details>

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2020-12-10-epub-a11y#section3) on 2020-12-10)

List of resolutions:

- ***[Resolution No. 2](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2020-12-10-epub-a11y#resolution2):  associate `CertifiedBy` with `CertifierCredentials` with refine.***

<details><summary><i>View the transcript</i></summary>

### 3. Clarify credentials need to be associated with certifier 

_See github issue [#1410](https://github.com/w3c/epub-specs/issues/1410)._




**Avneesh Singh:** credentials refine certifier, but this relationship is not clear in the spec.  
… Matt G suggested that we formalize this relationship.  

> *Matthew Chan:* Matt G's example [https://github.com/w3c/epub-specs/issues/1410#issuecomment-724095290](https://github.com/w3c/epub-specs/issues/1410#issuecomment-724095290).

> **Proposed resolution: associate `CertifiedBy` with `CertifierCredentials` with refine.** *(Avneesh Singh)*

> *Matthew Chan:* +1.

> *Charles LaPierre:* +1.

> *Ben Schroeter:* +1.

> *Juliette McShane:* +1.

> *Bill Kasdorf:* +1.

> *Jun’Ichi Yoshii:* +1.

> *George Kerscher:* +1.

> *Laura Brady:* +1.

> ***Resolution #2: associate `CertifiedBy` with `CertifierCredentials` with refine.***

</details>

**Comment by murata2makoto** [comment]
I have nothing against this resolution.  But I'm wondering if we should submit a defect report to ISO about this.  I certainly do not want to publish a JIS without mentioning the use of the refines attribute for associating CertifiedBy with CertifierCredentials.

**Comment by mattgarrish** [comment]
> But I'm wondering if we should submit a defect report to ISO about this.

Do we need to wait until this document becomes a recommendation to ensure the changes will become "official"?

**Comment by murata2makoto** [comment]
@mattgarrish No, we do not have to wait since the ISO/IEC version is not intended to be technically identical to any specification (including the W3C submission).  However, if EPUB A11Y 1.0 is going to be used (together with its own URI), we should ideally drop these two from the ISO/IEC version as well as the W3C submission.  But can we change the submission?
